python/servo/command_base.py

```diff
@@ -678,6 +678,7 @@ def build_env(self, hosts_file_path=None, target=None, is_build=False, test_unit
             # do not get precedence
             extra_path = [libpath] + extra_path
             extra_lib = [libpath] + extra_path
+            append_to_path_env(libpath, env, "LD_LIBRARY_PATH")
```

Comment:
This is already supposed to take place in the `if extra_lib` branch a few lines further down. Can we figure out why that isn't working?

Reply:
Oh I see what you mean. Shouldn't the setting be `extra_lib = [libpath] + extra_lib`?